articles/becoming-a-flutter-developer/index.md

```diff
@@ -0,0 +1,179 @@
+`Flutter` is an open-source UI software development kit that `Google` released in 2018. It uses Dart as its primary programming language. Flutter has led to the emergence of new software developers called `Flutter Developers`. 
+
+### Introduction
+
+Cross-platform applications enable companies to cut down on development costs. It eliminates the need to hire different software development teams. They also reduce the time and cost to manage different code bases by using a single code base to develop applications.  `Flutter developers` develop these cross-platform applications. 
+
+For these reasons, the need for Flutter Developers will continue to increase. 
+
+
+### Table of Content
+- [Who is a Flutter Developer](#who-is-a-flutter-developer)
+- [Roles and Responsibilites of a Flutter Developer](#roles-and-responsibilites-of-a-flutter-developer)
+- [Flutter Developer Tools](#flutter-developer-tools)
+- [Flutter Developer Skills](#flutter-developer-skills)
+- [Mistakes to Avoid When Learning Flutter Development](#mistakes-to-avoid-when-learning-flutter-development)
+- [Getting a job as a Flutter Developer](#getting-a-job-as-a-flutter-developer)
+   - [Job Description Example](#job-description-example)
+   - [Flutter Developer Interview Questions](#flutter-developer-interview-questions)
+- [Further Reading](#further-reading)
+- [Conclusion ](#conclusion)
+
+
+### Who is a Flutter Developer
+
+A Flutter developer is a software developer who develops cross-platform applications. They use `Flutter Framework` to develop these applications using a single base code. 
```

Comment:
`A Flutter developer is a software developer who develops cross-platform applications` Avoid repetition of sentences -- same as line 5